plex/provider_observer.py

```diff
@@ -21,15 +21,15 @@
 
 from lib.utils import log, mediaImport2str, mediaProvider2str
 
-ENDPOINT = '/:/websockets/notifications'
```

Comment:
Shouldn't this commit be squashed into bb9c6d57cba8cf60a95364846cf8081e94e50ba9 since you revert a change you did there?

Reply:
This still seems to be in the wrong commit.